h2o-py/h2o/estimators/targetencoder.py

```diff
@@ -26,7 +26,7 @@ class H2OTargetEncoderEstimator(H2OEstimator):
     algo = "targetencoder"
     supervised_learning = True
 
-    @deprecated_params({'k': 'inflection_point', 'f': 'smoothing', 'noise_level': 'noise'})
```

Comment:
@sebhrusen looks like the generator doesn't guarantee a stable order?

Reply:
nasty! will look at this.
how was it discovered? why the change in TE? does anyone really used all TE args as positional args? do people really enjoy unreadble code?

Reply:
actually the decorator leaves the positional args (when passed by the user) as-is, and only remaps named args, so I don't see what you mean exactly.

Reply:
@sebhrusen the issue is that in this PR target encoder was not touched (at least I don't see it) but we do have a different generated code

Reply:
ok ok, totally misunderstood! it's a generator issue, not a decorator issue :))

Reply:
ok, I see, I guess the generator is using a simple dict (not ordered), and different versions of Py may read the keys in different order.

Reply:
I can test a change and submit it to @koniecsveta to include it in this PR

Reply:
this was generated using Py2.7: it should actually fail in the PR build as this one uses Py3 (confirmation: http://mr-0xg1:8080/blue/organizations/jenkins/h2o-3-pipeline/detail/PR-5896/11/pipeline).
@koniecsveta I'll suggest you to just rebuild locally with Py3 before pushing, I'll fix the consistent ordering issue separately, thx.

Reply:
@sebhrusen I see - not worth the fix then - should we instead add a check in the build script that will require Python 3+?

Reply:
that's a good idea: we don't need to support 2.7 for the build phase

Reply:
thanks for clarification! I overlooked that I was building outside of my env for h2o3

Reply:
rebuilt here f163b9a